benchmarks/benchmarks/spatial.py

```diff
@@ -127,3 +127,40 @@ def time_count_neighbors(self, mn1n2, p, probe_radius, boxsize, leafsize):
         dim | # points T1 | # points T2 | p | probe radius |  BoxSize | LeafSize
         """
         self.T1.count_neighbors(self.T2, probe_radius, p=p)
+
+class SphericalVor(Benchmark):
+    params = [10, 100, 1000, 5000, 10000]
+    param_names = ['num_points']
+
+    def setup(self, num_points):
+        # generate uniform points on sphere (see:
+        # http://stackoverflow.com/a/23785326/2942522)
+        np.random.seed(123)
+        self.points = np.random.normal(size=(num_points, 3))
+        self.points /= np.linalg.norm(self.points, axis=1)[:, np.newaxis]
+
+    def teardown(self, num_points):
+        del self.points
```

Comment:
and the one below shouldn't be needed either. Did you see a problem without those?